spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/JvmMetricsAutoConfiguration.java

```diff
@@ -66,4 +69,10 @@ public ClassLoaderMetrics classLoaderMetrics() {
 		return new ClassLoaderMetrics();
 	}
 
+	@Bean
+	@ConditionalOnMissingBean
+	public DiskSpaceMetrics diskSpaceMetrics() {
+		return new DiskSpaceMetrics(new File(System.getProperty("user.dir")));
```

Comment:
Would we want to make this configurable? That could easily move it out of the "JVM" category if it was just some random path.

Reply:
Given how easy it is to expose it, I am not sure I would make it configurable (at least for a first version). However the disk space health indicator uses the current path (i.e. `new File(".")`) so I'd rather use that.